api-samples/tabs/inspector/tabs_api.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tabs Inspector</title>
+    <link rel="stylesheet" href="./tabs_api.css" />
+  </head>
+
+  <body>
+    <div id="windowList">
+      <!-- WindowItems Here -->
+    </div>
+    <template id="windowItem">
+      <div class="window-item">
+        <div class="window-id-wrapper">
+          Window: <span class="window_id"></span>
+        </div>
+        <div class="window-status-wrapper">
+          left:
+          <input class="window_left" type="text" /> top:
+          <input class="window_top" type="text" /> width:
+          <input class="window_width" type="text" /> height:
+          <input class="window_height" type="text" />
+          <input class="window_focused" type="checkbox" /> Focused
+          <input class="window_current" type="checkbox" /> Current
+          <button class="window_refresh">Refresh</button>
+        </div>
+        <div id="tabList">
+          <!-- TabItems Here -->
+        </div>
+        <button class="update_window_button">Update Window</button>
+        <button class="remove_window_button">Close Window</button>
+        <button class="refresh_selected_tab_button">
+          Refresh Selected Tab
+        </button>
+      </div>
+    </template>
+    <template id="tabItem">
+      <div class="tab-item">
+        <div class="wrapper">
+          <div class="tab_id"></div>
+          <div class="tab-actions-wrapper">
+            index:
+            <input type="text" class="tab_index" />
+            windowId:
+            <input type="text" class="tab_window_id" />
+            <button class="move_tab_button">Move</button>
+            <button class="refresh_tab_button">Refresh</button>
+          </div>
+        </div>
+        <div class="wrapper">
+          <div>
+            <div class="label">title:</div>
+            <input type="text" class="tab_title" />
+          </div>
+          <div>
+            <div class="label">url:</div>
+            <input type="text" class="tab_url" />
+          </div>
+          <div><input type="checkbox" class="tab_selected" /> Selected</div>
```

Comment:
I think this would make more sense as "active", right? And should we make the checkbox readonly?